sqldelight-studio-plugin/testData/formatting/Formatting_after.sq

```diff
@@ -0,0 +1,33 @@
+import java.util.List;
+import java.util.ArrayList;
+
+CREATE TABLE test (
+    _id INTEGER NOT NULL PRIMARY KEY,
+    abs_list BLOB AS List NOT NULL,
+    array_list BLOB AS ArrayList
+);
+
+some_select:
+SELECT * FROM test;
+
+/** This is
```

Comment:
Works for me

On Wed, Aug 3, 2016 at 12:41 PM Alec Strong <notifications@github.com>
wrote:

> In sqldelight-studio-plugin/testData/formatting/Formatting_after.sq
> <https://github.com/square/sqldelight/pull/431#discussion_r73355403>:
>
> > @@ -0,0 +1,33 @@
> > +import java.util.List;
> > +import java.util.ArrayList;
> > +
> > +CREATE TABLE test (
> > +    _id INTEGER NOT NULL PRIMARY KEY,
> > +    abs_list BLOB AS List NOT NULL,
> > +    array_list BLOB AS ArrayList
> > +);
> > +
> > +some_select:
> > +SELECT * FROM test;
> > +
> > +/** This is
>
> alright so this is actually way harder than i originally though, since
> you're not just changing spacing anymore you're adding a new character (the
> * at the beginning of the new line). The java formatter has this class
> InjectedLanguageBlockBuilder to handle it since you have to edit the AST
> tree but i dont want to get into that yet
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/square/sqldelight/pull/431/files/79fc87a44b49f376bf12b37575870ff7bd4c6c97#r73355403>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAEEEcIJPePSt_vN99bEhpzw_-ffjVUNks5qcLw-gaJpZM4JXkxr>
> .
>